src/__tests__/redux/actions/profile.test.js

```diff
@@ -0,0 +1,80 @@
+import configureMockStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import mockAxios from 'axios';
+import * as actions from '../../../redux/actions/profile';
+import * as types from '../../../redux/actions/actionTypes';
+
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+
+describe('view profile actions', () => {
+  it('handles viewProfile', () => {
+    const profile = 'profile';
+    const expectedAction = {
+      type: types.VIEW_PROFILE,
+      payload: profile,
+    };
+
+    expect(actions.getProfile(profile)).toEqual(expectedAction);
+  });
+});
+
+
+describe('handles getProfileRequest', () => {
```

Comment:
@Dubby20 this is not a proper describe title

Reply:
Thanks. I will change it